scripts/parser-tests/test262/allowlist.txt

```diff
@@ -6,9 +6,22 @@ language/import/import-attributes/json-named-bindings.js(default)
 language/import/import-attributes/json-named-bindings.js(strict mode)
 
 
-# 4 valid programs produced a parsing error
+# 17 valid programs produced a parsing error
 
-language/import/import-defer/errors/resolution-error/import-defer-of-missing-module-fails.js(default)
-language/import/import-defer/errors/resolution-error/import-defer-of-missing-module-fails.js(strict mode)
-language/import/import-defer/errors/syntax-error/import-defer-of-syntax-error-fails.js(default)
-language/import/import-defer/errors/syntax-error/import-defer-of-syntax-error-fails.js(strict mode)
+staging/sm/String/make-normalize-generateddata-input.py(default)
+staging/sm/String/make-normalize-generateddata-input.py(strict mode)
+staging/sm/extensions/inc-dec-functioncall.js(default)
+staging/sm/lexical-environment/block-scoped-functions-deprecated-redecl.js(default)
+staging/sm/lexical-environment/block-scoped-functions-hoisted-tdz.js(default)
+staging/sm/lexical-environment/var-in-catch-body-annex-b-eval.js(default)
+staging/sm/module/bug1693261-async.mjs(default)
+staging/sm/module/bug1693261-async.mjs(strict mode)
+staging/sm/module/bug1693261-c1.mjs(default)
+staging/sm/module/bug1693261-c1.mjs(strict mode)
+staging/sm/module/bug1693261-c2.mjs(default)
+staging/sm/module/bug1693261-c2.mjs(strict mode)
+staging/sm/module/bug1693261-x.mjs(default)
+staging/sm/module/bug1693261-x.mjs(strict mode)
+staging/sm/regress/regress-563221.js(default)
+staging/sm/regress/regress-609617.js(default)
+staging/sm/statements/for-of-iterator-close.js(default)
```

Comment:
This last one is a Babel bug, the rest are just broken tests

Reply:
#4345, #4346, #4347

Reply:
https://github.com/babel/babel/pull/17021/files#diff-e1ef5153354cf3d7e9b62bf220450240b6e16c6535eaee694e61043ab544d110R11-R12
There seem to be two `.py` files, do you know what the reason is?

Reply:
I just asked to the maintainers. I think it's a bug in test262 (all files except for those containing `_FIXTURE` are meant to be interpreted as JS tests), but I'll wait for them to answer.